Algorithm/QCAlgorithm.cs

```diff
@@ -1060,9 +1060,16 @@ public void SetBenchmark(string symbol)
             // check existence
             symbol = symbol.ToUpper();
             var security = Securities.FirstOrDefault(x => x.Key.Value == symbol).Value;
-            _benchmarkSymbol = security == null
-                ? QuantConnect.Symbol.Create(symbol, SecurityType.Equity, Market.USA)
-                : security.Symbol;
+            if (security == null)
+            {
+                Debug($"Warning: SetBenchmark({symbol}): no existing security found, benchmark security will be added with {SecurityType.Equity} type.");
```

Comment:
We could add the information that there is a method overload that accepts the `SecurityType`: `SetBenchmark(SecurityType, string)`.